docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,87 @@
+---
+slug: /serverless/security/automatic-import
+title: Automatic Import
+description: Use Automatic Import to quickly ingest third-party data.
```

Comment:
Could add `normalize` here, as it's an important aspect. 

Use Automatic Import to quickly normalize and ingest third-party data.